Code releases 05.20/d8e84299-d99b-40e5-a34d-791101435ee6.md

```diff
@@ -1,5 +1,39 @@
-Super Attributes in the Spryker Commerce OS are used to distinguish between the different Product Variants of an abstract product. Super Attributes define each Concrete Product and can consist of whichever distinguishing feature you wish to highlight, such as size or color. This information can either be manually managed or imported and processed automatically.
+The super attribute is the only way to tell Spryker, that a product contains multiple variants (or concrete products) to allow your customer to select the variant they are looking for.
+Spryker supports an unlimited amount of super attributes for a given abstract product. See below to understand how abstract and concrete products works with super attributes.
 
-Consider the example of a T-shirt, the Abstract Product, that is available in the sizes small, medium, and large, three different Product Variants. The Abstract Product "T-shirt" appears as a search result. On the product detail page the customer can select between the product variants small, medium, large and put it into the cart.
+## Hierarchy of products
 
-![image](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Product+Abstraction/product-abstraction.png){height="" width=""}
\ No newline at end of file
+In Spryker, you have 2 product levels:
```

Comment:
what about `product group` ?

Reply:
Product group is something different to group abstract products via the color attributes. Since here we talk only about super attributes, this information will go into another documentation such as https://documentation.spryker.com/v4/docs/product-group